Marlin/ultralcd.cpp

```diff
@@ -670,7 +670,7 @@ static void lcd_move_menu_axis();
 
 static void _lcd_move(const char *name, int axis, int min, int max) {
   if (encoderPosition != 0) {
-    refresh_cmd_timeout();
+    void refresh_cmd_timeout();
```

Comment:
Wrong line. The declaration should be changed in `Marlin.h`, not this function call.